interchain/interchain-tests/src/bin/ibc_hook.rs

```diff
@@ -159,17 +158,73 @@ pub fn test_pfm() -> AnyResult<()> {
     Ok(())
 }
 
-pub fn init_counter<Chain: CwEnv>(
-    chain: Chain,
-) -> AnyResult<counter_contract::CounterContract<Chain>> {
-    let counter = counter_contract::CounterContract::new(chain);
-    counter.upload()?;
-    counter.instantiate(
-        &counter_contract::msg::InstantiateMsg { count: 0 },
-        None,
-        None,
-    )?;
-    Ok(counter)
+// Counter contract doesn't use crates.io version of cw-orch, some weird hacks here.
```

Comment:
So this is why you wanted to publish the counter contract? 

We could still do that. I don't think having this code here is great.

Reply:
Yeah! Another clean option would be to make separate workspace for contracts in cw-orch repo with patch section

Reply:
> Yeah! Another clean option would be to make separate workspace for contracts in cw-orch repo with patch section

Can we do this before merging this PR? Otherwise we'll have this "dead" code in the repo.

Reply:
Done! https://github.com/AbstractSDK/cw-orchestrator/pull/468

Reply:
Should we wait with merging until we've switched to cw-orch 0.25?

Because I don't think that PR is included in the 0.24 release tag.

Reply:
Yep tests won't run until we use latest starship